README.rst

```diff
@@ -29,3 +29,19 @@ To use:
 * ``pipenv sync``
 
 * ``pipenv run upload``
+
+
+Troubleshooting
+***************
+
+The overall flow looks like the following:
+
+* No 2FA may be a must.
+
+* If your uploading device is the 1st time to upload, or your last uploading is too long ago, you may need a SMS validation for your device because of security concern.
+
+* The corresponding credential json may need to update (by the channel owner of youtube/gmail account)
+
+* This app needs approval by the channel owner's youtube/gmail account (via web browser by clicking the authorization link).
+
+* In 2020 we are aware that Google Security Team will review your uploaded videos via your customized application. The uploaded videos is "private(locked" by default and not allowed to set as "public" manually until the approval of Google Security Team.
```

Comment:
There is a missing ")" in `"private(locked"`.

Reply:
Nice catch!! : D